beluga_demo_nav2/launch/navigation_launch.py

```diff
@@ -0,0 +1,145 @@
+        default_value='nav2_container',
+        description='the name of container that'
+        'nodes will load in if use composition',
+    )
+
+    load_composable_nodes = LoadComposableNodes(
+        target_container=container_name,
+        composable_node_descriptions=[
+            ComposableNode(
+                package='nav2_controller',
+                plugin='nav2_controller::ControllerServer',
+                name='controller_server',
+                parameters=[configured_params],
+                remappings=[('cmd_vel', 'cmd_vel_nav')],
+            ),
+            ComposableNode(
+                package='nav2_smoother',
+                plugin='nav2_smoother::SmootherServer',
+                name='smoother_server',
+                parameters=[configured_params],
+            ),
+            ComposableNode(
+                package='nav2_planner',
+                plugin='nav2_planner::PlannerServer',
+                name='planner_server',
+                parameters=[configured_params],
+            ),
+            ComposableNode(
+                package='nav2_behaviors',
+                plugin='behavior_server::BehaviorServer',
+                name='behavior_server',
+                parameters=[configured_params],
+            ),
+            ComposableNode(
+                package='nav2_bt_navigator',
+                plugin='nav2_bt_navigator::BtNavigator',
+                name='bt_navigator',
+                parameters=[configured_params],
+            ),
+            ComposableNode(
+                package='nav2_waypoint_follower',
+                plugin='nav2_waypoint_follower::WaypointFollower',
+                name='waypoint_follower',
+                parameters=[configured_params],
+            ),
+            ComposableNode(
+                package='nav2_velocity_smoother',
+                plugin='nav2_velocity_smoother::VelocitySmoother',
+                name='velocity_smoother',
+                parameters=[configured_params],
+                remappings=[
+                    ('cmd_vel', 'cmd_vel_nav'),
+                    ('cmd_vel_smoothed', 'cmd_vel'),
+                ],
+            ),
+            ComposableNode(
+                package='nav2_lifecycle_manager',
+                plugin='nav2_lifecycle_manager::LifecycleManager',
+                name='lifecycle_manager_navigation',
+                parameters=[
+                    {
+                        'use_sim_time': True,
+                        'autostart': True,
+                        'node_names': lifecycle_nodes,
+                    }
+                ],
+            ),
+        ],
+    )
+
+    # Create the launch description and populate
+    ld = LaunchDescription()
+
+    # Declare the launch options
+    ld.add_action(declare_params_file_cmd)
+    ld.add_action(declare_container_name_cmd)
+
+    # Add the actions to launch all of the navigation nodes
+    ld.add_action(load_composable_nodes)
+
+    return ld
```

Comment:
@Alondruck meta: why not simplify the launchfiles hierarchy a bit? Do we need waypoint following support to show Beluga can successfully fill in for Nav2 AMCL?

Reply:
I agree that I can remove waypoint following from here. Anyway, I'm not too concerned about this file because the focus of the tutorial is on `localization_launch.py` where we configure to use `beluga_amcl`. The rest is literally a simplified copy of nav2 because it is assumed that if someone has reached this tutorial, they have previously completed the getting started tutorial for nav2.